src/components/UserMenu/components/AvatarMyself.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import { useQuery, models } from 'cozy-client'
+import Avatar from 'cozy-ui/transpiled/react/Avatar'
+import Skeleton from 'cozy-ui/transpiled/react/Skeleton'
+
+import { buildMyselfQuery } from 'lib/queries'
+
+const { getInitials } = models.contact
```

Comment:
nit: deep imports `import { getInitials } from 'cozy-client/dist/models/contact'` are no longer required?

Reply:
It looks safer for tree shaking I updated it (but did not test if it is better for now)